tatianamac/selfdefined
# Style fixes

Hello folks!

**Description**
This PR has a parcel of assorted style changes. Some themes are: Changing classnames to be more [BEM-compliant](http://getbem.com/introduction/), refactoring styles to DRY, and addressing homepage layout issues.

**Testing steps:**

1. Navigate to the homepage
2. Resize the screen width between 800-1200px. Ensure the text fills the screen pleasingly
3. Resize screen to 320px (iPhone 5)
4. Ensure the Twitter handle in the "Help" box in the homepage introduction doesn't overflow its container.
